pkg/addons/applier.go

```diff
@@ -319,37 +322,29 @@ func (a *Applier) Versions(additionalCharts []v1beta1.Chart) (map[string]string,
 	return versions, nil
 }
 
-// waitForKubernetes waits until we manage to make a successful connection to the
-// Kubernetes API server.
-func (a *Applier) waitForKubernetes(ctx context.Context) error {
+// waitForKubernetes waits for codedns and metrics-server to be ready in kube-system.
+func (a *Applier) waitForKubernetes(ctx context.Context, cli client.Client) error {
```

Comment:
this function was previously unused, and the only bit that was retained is the name